isis/tests/FunctionalTestsApollocal.cpp

```diff
@@ -29,10 +30,18 @@ TEST_F(ApolloCube, FunctionalTestApollocalDefault) {
 
   Cube oCube(outCubeFileName, "r");
 
-  std::unique_ptr<Histogram> oCubeStats(oCube.histogram());
-
-  EXPECT_NEAR(oCubeStats->Average(), -124.188,          0.001);
-  EXPECT_NEAR(oCubeStats->Sum(), -65125546242.836,   0.001);
-  EXPECT_NEAR(oCubeStats->ValidPixels(), 524410000,     0.001);
-  EXPECT_NEAR(oCubeStats->StandardDeviation(), 1056.736, 0.001);
+  // Check a region with both Null and non-Null data
+  Brick brick(reseauSize + 10,reseauSize + 10,1,oCube.pixelType());
+  int baseSamp = (int)(reseaus[0].first+0.5) - (reseauSize/2);
+  int baseLine = (int)(reseaus[0].second+0.5) - (reseauSize/2);
+  brick.SetBasePosition(baseSamp,baseLine,1);
+  oCube.read(brick);
+  Statistics reseauStats;
+  reseauStats.AddData(&brick[0], brick.size());
+
+  EXPECT_NEAR(reseauStats.Average(), -2864.497, 0.001);
+  EXPECT_NEAR(reseauStats.Sum(), -30389453.463, 0.001);
+  EXPECT_EQ(reseauStats.ValidPixels(), 10609);
+  EXPECT_EQ(reseauStats.NullPixels(), 2160);
```

Comment:
Similar to my other comment on valid vs null pixels. These numbers seem a little off based on my assumptions of how the reseauses are treated in ISIS. Just want to clarify why this is

Reply:
The brick for this test is a 10 pixel bigger than the Reseaus so that we can get a region that has null and non-null data. So, that's why we get the 103x103 reseau data but then a bunch of nulls, (103*10) * 2 + 10*10 = 2160. Part of the functionality of apollocal is that it converts any special pixels in the input to Nulls, so I wanted to check that.